kernel_install

```diff
@@ -38,9 +38,32 @@ kernel_revision="$(echo "${kernel_ext}" | cut -d '-' -f 3)"
 boot_name="Gentoo ${kernel_version}"
 [ "${kernel_tag}" != 'gentoo' ] && boot_name="${boot_name} ${kernel_tag}"
 
+efi_ro=false
+if mount | grep "^efivarfs" | grep "ro"
+then
+	efi_ro=true
+	efi_mpoint=$(mount | grep "^efivarfs" | cut -d " " -f 3)
+fi
+
+while $efi_ro; do
```

Comment:
Rational for having a prompt? Wouldn't it be more straight forward to just remount it if read-only?

Reply:
The idea was that it was mounted 'ro' for a reason which explain the disclaimer. I'll change it if you want.

Reply:
Hum, I think that as long as we already mount `/boot`, we can safely assume that the user would also want `efivarfs` to be remounted as needed. So yeah, no prompt, assume that the user want to have it remounted.